wiring/inc/spark_wiring_ble.h

```diff
@@ -1010,6 +1016,9 @@ class BleLocalDevice {
     ssize_t getScanResponseData(BleAdvertisingData* scanResponse) const;
     ssize_t getScanResponseData(BleAdvertisingData& scanResponse) const;
 
+    int setAdvertisingScheme(BleAdvertisingScheme scheme) const;
+    int getAdvertisingScheme(BleAdvertisingScheme* scheme) const;
```

Comment:
Similar to other cases let's add both pointer and reference getter perhaps?
```
int getScanParameters(BleScanParams* params) const;
int getScanParameters(BleScanParams& params) const;
```

Reply:
Sounds good.